getting-started/openshift/installation.md

```diff
@@ -85,6 +85,18 @@ curl {{ "/manifests/ocp/tigera-operator/02-tigera-operator.yaml" | absolute_url
 curl {{ "/manifests/ocp/01-cr-installation.yaml" | absolute_url }} -o manifests/01-cr-installation.yaml
 ```
 
+#### Add install time resources
+
+Add any [Calico resources]({{site.baseurl}}/reference/resources) that need to be created at startup to
```

Comment:
I don't think that this section is worded quite right for the average user. 

I think we either want a separate "How to provide install-time configuration" document, or we need to make it so that it's obvious to the reader whether or not they care about this section. 

What about something like this:

-----------------------------------------------------------------------


#### Optionally provide additional configuration

You may want to provide Calico with additional configuration at install-time. For example, BGP configuration or peers. When installing on OpenShift, you can use a Kubernetes ConfigMap with your desired [Calico resources]() in order to set configuration as part of the installation. if you do not need to provide additional configuration, you can skip this section.

To include resources during installation, edit `manifests/02-configmap-calicoctl-resources.yaml` in order to add your own configuration. 

You can also create the configmap from a directory, where each file within is a Calico resource. 

...